local-scripts/print-json.py

```diff
@@ -14,6 +14,14 @@
 def set_file_indent(file):
     # This code breaks in 2030 and.or after we assign 1 million GSDs per year
     file_name = os.path.basename(file)
+
+    # We match 2021 through 2029, 1 million and up so that's guaranteed GSD space only
+    # GSD bot uses 2 space for indent
+    # The CVE Bot uses the default 4 spaces, hence the need for checking
+    # We can ismply read the file, scheck the second line and count spaces because
+    # some workflow (e.g. linux updates) involves using the command line jq which
```

Comment:
some workflows involve using

Also what's jq?

Reply:
https://stedolan.github.io/jq/

Reply:
Nice, can you include that link in the comment so that it doesn't require institutional knowledge for future devs looking at the code?

some **workflow** (e.g. linux updates) **involves** using the command line jq which
some **workflows** (e.g. linux updates) **involve** using the command line jq which

Bold for clarity on what needs changed.

Reply:
ok should be good now